app/helpers/model_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module ModelHelpers
```

Comment:
@case-eee Thank you for the comment! I must have misunderstood the purpose of the helper. It wasn't very clear that it is for `views` only. Do you have another suggestion?
@akintner